neuralop/models/uno.py

```diff
@@ -153,19 +153,25 @@ def __init__(self,
         self.separable = separable
         self.preactivation = preactivation
         self._incremental_n_modes = incremental_n_modes
-        self.output_scaling_factor = output_scaling_factor
         self.operator_block = operator_block
         self.integral_operator = integral_operator
         
         if self.horizontal_skips_map is None:
             self.horizontal_skips_map = {}
             for i in range(n_layers//2,0,):
                 self.horizontal_skips_map[n_layers - i -1] = i
-                
+
+        self.output_scaling_factor = np.ones_like(self.uno_scalings[0])
+        for k in self.uno_scalings:
+            self.output_scaling_factor = np.multiply(self.output_scaling_factor, k)
+        self.output_scaling_factor = self.output_scaling_factor.tolist()
+        if len(self.output_scaling_factor) == 1:
+            self.output_scaling_factor = self.output_scaling_factor[0]
+
         if isinstance(self.output_scaling_factor, (float, int)):
             self.output_scaling_factor = [self.output_scaling_factor]*self.n_dim
         
-
+        print("calculated out factor", self.output_scaling_factor)
```

Comment:
Let's add an attribute `verbose` (pass it in init) and only print `if self.verbose:`

Reply:
done